package.json

```
@@ -64,7 +64,8 @@
     "postinstall": "patch-package",
     "prepack": "pinst --disable",
     "postpack": "pinst --enable",
-    "removedb": "rm -rf .entropy && docker compose --file dev/docker-scripts/two-nodes.yaml down 2> /dev/null"
+    "publish:rc": "yarn burn && yarn build && npm publish --no-tag",
```

```suggestion
    "publish:rc": "yarn burn && yarn build && npm publish --tag next",
```

https://docs.npmjs.com/cli/v10/commands/npm-publish#tag